src/test/java/com/myFirstProject/myFirstProject/service/AccountServiceImplTest.java

```diff
@@ -76,7 +78,7 @@ public void updateAccountIfItNotNull() {
     }
 
     private Account buildAccount() {
-        Account account = buildNewAccount(BigDecimal.ONE);
+        Account account = buildNewAccount(BigDecimal.TEN);
```

Comment:
buildNewAccount - strange name. Can we build oldAccount???

Reply:
It is good name

Reply:
it is bad name, it should be just buildAccount

Reply:
i have such method build account, it descriebes different situations

Reply:
done